openstack/limes/templates/configmap.yaml

```
@@ -11,6 +11,15 @@ data:
       - {{ . }}
     {{- end }}
 {{ toYaml .Values.limes.clusters.ccloud | indent 4 }}
+    mail_notifications:
+      endpoint: {{ .Values.limes.clusters.ccloud.catalog_url | replace "limes-3" "limes-campfire" }}
```

Docs require that there is a trailing slash, so we should play it safe:
```suggestion
      endpoint: {{ .Values.limes.clusters.ccloud.catalog_url | replace "limes-3" "limes-campfire" | trimSuffix "/" }}/
```

The trimSuffix is to ensure there is no double trailing slash if the URL already had one before.